app/server.go

```diff
@@ -781,6 +791,46 @@ func doSessionCleanup(s *Server) {
 	s.Store.Session().Cleanup(model.GetMillis(), SESSIONS_CLEANUP_BATCH_SIZE)
 }
 
+func doLicenseExpirationCheck(s *Server) {
+	s.FakeApp().LoadLicense()
+	license := s.License()
+
+	if license == nil {
+		mlog.Info("License cannot be found.")
```

Comment:
I'm not sure about logging this cases and the next one. For me that 2 cases can be "business as usual".

Reply:
Yes, I think Debug is more appropriate here, because this information is not very useful for regular users without license, until it is deactivated, and when that happens, is properly logged.